notification/notification-center-item.jsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-env es6, browser */
+
+"use strict";
+
+const React = require("react"),
+	  format = require("../lib/format.js");
+
+module.exports = (core, ...args) => {
+	const NotificationItem = require("./notification-item.es6")(core, ...args);
+
+	class NotificationCenterItem extends React.Component {
+		constructor(props) {
+			super(props);
+
+			this.state = { item: new NotificationItem(props.note) };
```

Comment:
Good approach to making code more OO where needed. We can use this also for user and room... keep plain objects in "public" places and create class instances in "private" code.